src/gui/tray/Window.qml

```diff
@@ -99,7 +99,9 @@ Window {
         anchors.fill:   parent
         radius:         Style.trayWindowRadius
         border.width:   Style.trayWindowBorderWidth
-        border.color:   Style.menuBorder
+        border.color:   Style.windowText
+
+        color: Style.window

```

Comment:
@FlexW You sure this is not `Style.windowText`?

Reply:
Yes im sure. Why would you choose windowText? Window is the background color.